features/FEATURE_PSA/supporting_psa_in_mbed-os.md

```diff
@@ -120,8 +183,36 @@ example mbed_app.json:
 }
 ```
 
+# Building TF-M
+On dual v7-M and v8-M targets, TF-M will be running on SPE providing PSA services. A python script `tools/psa/build_tfm.py` automates building TF-M and copying the binary to predefined location defined by attribute `delivery_dir`.
 
-## New directory structure
+Usage:
+```console
+python3 tools/psa/build_tfm.py
+```
+Supported options:
+```console
+usage: build_tfm.py [-h] [-m {CY8CPROTO_064_SB_S}] [-t {ARMCLANG,GNUARM}] [-d]
+                    [-l] [--clone] [--commit] [--clean-build] [-v]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -m {CY8CPROTO_064_SB_S}, --mcu {CY8CPROTO_064_SB_S}
+                        Build for the given MCU
+  -t {ARMCLANG,GNUARM}, --toolchain {ARMCLANG,GNUARM}
+                        Build for the given tool chain (default is
+                        tfm_default_toolchain)
+  -d, --debug           Set build profile to debug
+  -l, --list            Print supported TF-M secure targets
+  --clone               Clone TF-M git repo and its dependencies
+  --commit              Create a git commit for each platform
+  --clean-build         Remove cmake build directory
```

Comment:
For all uses of `cmake` that are supposed to be for humans:

```diff
- cmake
+ CMake
```

Reply:
Fixed

Reply:
```diff
- Remove cmake build directory
+ Remove CMake build directory
```